modules/react/button/stories/button/examples/Primary.tsx

```diff
@@ -5,11 +5,17 @@ import {HStack} from '@workday/canvas-kit-react/layout';
 import {plusIcon} from '@workday/canvas-system-icons-web';
 
 export const Primary = props => (
-  <HStack spacing="s" padding="s">
-    <PrimaryButton {...props}>Primary</PrimaryButton>
-    <p>{JSON.stringify(props.theme)}</p>
-    <PrimaryButton icon={plusIcon} iconPosition="right" {...props}>
-      Primary
-    </PrimaryButton>
-  </HStack>
+  <>
+    <HStack spacing="s" padding="s">
+      <PrimaryButton {...props}>Primary</PrimaryButton>
+      <p>{JSON.stringify(props.theme)}</p>
+      <PrimaryButton icon={plusIcon} iconPosition="start" {...props}>
+        Primary
+      </PrimaryButton>
+      <PrimaryButton icon={plusIcon} iconPosition="end" {...props}>
+        Primary
+      </PrimaryButton>
+      <PrimaryButton icon={plusIcon} {...props} />
+    </HStack>
+  </>
```

Comment:
Passing top-level props around in story examples will blow up Gatsby and the docs site. I really thought we resolved this a while back? What's below will fix it.

```tsx
export const Primary = () => (
  <HStack spacing="s" padding="s">
    <PrimaryButton>Primary</PrimaryButton>
    <PrimaryButton icon={plusIcon} iconPosition="start">
      Primary
    </PrimaryButton>
    <PrimaryButton icon={plusIcon} iconPosition="end">
      Primary
    </PrimaryButton>
    <PrimaryButton icon={plusIcon} />
  </HStack>
)
```